Makefile

```diff
@@ -7,6 +7,12 @@ else
     ENV_ARG	 := dev
 endif
 
+ifdef clam
+	FOREMAN_ARG := all=1
+else
+	FOREMAN_ARG := all=1,clamd=0,freshclam=0
```

Comment:
Is this the same as web=1, cron=1, clamd=0, freshclam=0 ?

Reply:
Yep -- its just the config we had before (and similar to production)